packaging/generic-rpm/amazon-ecs-init.spec

```diff
@@ -42,7 +42,6 @@ Source4:        https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-arm64-v%{bun
 Source5:        amazon-ecs-volume-plugin.service
 Source6:        amazon-ecs-volume-plugin.socket
 
-BuildRequires:  golang >= 1.7
```

Comment:
Is this changed because the build is done with 1.17.2?

Reply:
Yes.  This will be updated as we start testing our builds in Koji.